src/counterResults.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { getValueInput } = require('./getValueInput.js');
+
+function counterResults(input, random) {
```

Comment:
Consider using more descriptive variable names, like 'userInput' and 'randomNumber' instead of 'input' and 'random'